{"links": {"self": {"href": "data/repositories/osrf/subt/issues/74/comments/52613284.json"}, "html": {"href": "#!/osrf/subt/issues/74#comment-52613284"}}, "issue": {"links": {"self": {"href": "data/repositories/osrf/subt/issues/74.json"}}, "type": "issue", "id": 74, "repository": {"links": {"self": {"href": "data/repositories/osrf/subt.json"}, "html": {"href": "#!/osrf/subt"}, "avatar": {"href": "data/bytebucket.org/ravatar/{3c95f6ad-c304-407c-b838-09597d836552}ts=2272898"}}, "type": "repository", "name": "subt", "full_name": "osrf/subt", "uuid": "{3c95f6ad-c304-407c-b838-09597d836552}"}, "title": "Is the new simulator working for anyone?"}, "content": {"raw": "Hi @Alfredo Bencomo, \n\nI did an install using docker and I get similar problems as described by everyone else. \n\n1. If I follow step 4 to test the SubT Installation \u201c`ign launch virtual_stix.ign -v 4\u201d` then I can see the whole cave without errors. \n2. If I follow the exampleSetup with \u201c`ign launch competition.ign -v 4\u201d` and then \u201c`ign launch team.ign\u201d`, I get the white screen mentioned. \n3. Finally if I follow your last suggestion of \n\n\u200c    source /opt/ros/melodic/setup.bash\n\nsource ~/subt\\_ws/install/setup.bash\n\ncd ~/subt\\_ws/\n\nign launch -v 4 virtual\\_stix.ign robotName1:=X1 robotConfig1:=X1\\_SENSOR\\_CONFIG\\_1 robotName2:=X2 robotConfig2:=X2\\_SENSOR\\_CONFIG\\_1 robotName3:=X3 robotConfig3:=X3\\_SENSOR\\_CONFIG\\_1 robotName4:=X4 robotConfig4:=X4\\_SENSOR\\_CONFIG\\_1\n\nThen the world gets loaded but no robots. And I see the following errors: \n\n\\[Err\\] \\[[LevelManager.cc:134](http://LevelManager.cc:134)\\] Could not find a plugin tag with name ignition::gazebo. Levels and distributed simulation will not work.\n\nUnable to read \\[/home/developer/subt\\_ws/install/share/subt\\_ign/worlds/virtual\\_stix.dot\\] file  \n\\[Err\\] \\[[VisibilityTable.cc:57](http://VisibilityTable.cc:57)\\] Unable to populate visibility graph using path\\[/home/developer/subt\\_ws/install/share/subt\\_ign/worlds/virtual\\_stix.dot\\]  \n\\[Err\\] \\[[VisibilityRfModel.cc:36](http://VisibilityRfModel.cc:36)\\] Unable to load visibility table data files\n\nand the following shows 3 times\n\n\\[Err\\] \\[[GazeboFactory.cc:175](http://GazeboFactory.cc:175)\\] Factory service call timed out.", "markup": "markdown", "html": "<p>Hi @Alfredo Bencomo, </p>\n<p>I did an install using docker and I get similar problems as described by everyone else. </p>\n<ol>\n<li>If I follow step 4 to test the SubT Installation \u201c<code>ign launch virtual_stix.ign -v 4\u201d</code> then I can see the whole cave without errors. </li>\n<li>If I follow the exampleSetup with \u201c<code>ign launch competition.ign -v 4\u201d</code> and then \u201c<code>ign launch team.ign\u201d</code>, I get the white screen mentioned. </li>\n<li>Finally if I follow your last suggestion of </li>\n</ol>\n<p>\u200c    source /opt/ros/melodic/setup.bash</p>\n<p>source ~/subt_ws/install/setup.bash</p>\n<p>cd ~/subt_ws/</p>\n<p>ign launch -v 4 virtual_stix.ign robotName1:=X1 robotConfig1:=X1_SENSOR_CONFIG_1 robotName2:=X2 robotConfig2:=X2_SENSOR_CONFIG_1 robotName3:=X3 robotConfig3:=X3_SENSOR_CONFIG_1 robotName4:=X4 robotConfig4:=X4_SENSOR_CONFIG_1</p>\n<p>Then the world gets loaded but no robots. And I see the following errors: </p>\n<p>[Err] [<a data-is-external-link=\"true\" href=\"http://LevelManager.cc:134\" rel=\"nofollow\">LevelManager.cc:134</a>] Could not find a plugin tag with name ignition::gazebo. Levels and distributed simulation will not work.</p>\n<p>Unable to read [/home/developer/subt_ws/install/share/subt_ign/worlds/virtual_stix.dot] file<br />\n[Err] [<a data-is-external-link=\"true\" href=\"http://VisibilityTable.cc:57\" rel=\"nofollow\">VisibilityTable.cc:57</a>] Unable to populate visibility graph using path[/home/developer/subt_ws/install/share/subt_ign/worlds/virtual_stix.dot]<br />\n[Err] [<a data-is-external-link=\"true\" href=\"http://VisibilityRfModel.cc:36\" rel=\"nofollow\">VisibilityRfModel.cc:36</a>] Unable to load visibility table data files</p>\n<p>and the following shows 3 times</p>\n<p>[Err] [<a data-is-external-link=\"true\" href=\"http://GazeboFactory.cc:175\" rel=\"nofollow\">GazeboFactory.cc:175</a>] Factory service call timed out.</p>", "type": "rendered"}, "created_on": "2019-06-17T10:06:13.939439+00:00", "user": {"display_name": "Hector Escobar", "uuid": "{1c140a49-f2c5-4727-abd9-eb427bfe4253}", "links": {"self": {"href": "https://api.bitbucket.org/2.0/users/%7B1c140a49-f2c5-4727-abd9-eb427bfe4253%7D"}, "html": {"href": "https://bitbucket.org/%7B1c140a49-f2c5-4727-abd9-eb427bfe4253%7D/"}, "avatar": {"href": "data/secure.gravatar.com/avatar/8da90eb0e9b47c9adfc2893e5190483fd=httpsavatar-management--avatars.us-west-2.prod.public.atl-paas.netinitialsHE-1.png"}}, "nickname": "hector_escobar", "type": "user", "account_id": "5b5b1c941394ae6a72c021df"}, "updated_on": null, "type": "issue_comment", "id": 52613284}